base_user_role/models/role.py

```diff
@@ -74,6 +74,7 @@ class ResUsersRoleLine(models.Model):
     _name = "res.users.role.line"
     _description = "Users associated to a role"
 
+    user_active = fields.Boolean(related="user_id.active")
```

Comment:
why don't you call this field `active`? Then you don't need any other change and Odoo standard treatment of the active flag will do what you want.

Reply:
Perfect, I have applied the changes.
Thanks for the suggestion! :+1: